ipaplatform/redhat/paths.py

```diff
@@ -28,6 +28,11 @@
 
 # Fallback to default path definitions
 from ipaplatform.base.paths import BasePathNamespace
+from ipaplatform.osinfo import osinfo
+
+# RHEL 7 and earlier use /etc/sysconfig/nfs
+# RHEL 8 uses /etc/nfs.conf
+HAS_NFS_CONF = osinfo.id == 'rhel' and int(float(osinfo.version_id)) >= 8
```

Comment:
Why can't you import `HAS_NFS_CONF` from the constants instead? It is better to have the check in a single place.

Reply:
Done, thanks @abbra